Lib/test/test_tarfile.py

```diff
@@ -1368,7 +1368,7 @@ def write(self, data):
 
             f = BadFile()
             with self.assertRaises(exctype):
-                tar = tarfile.open(tmpname, self.mode, fileobj=f,
+                tarfile.open(tmpname, self.mode, fileobj=f,
                                    format=tarfile.PAX_FORMAT,
                                    pax_headers={'non': 'empty'})
```

Comment:
These two arguments should be indented under `tmpname`.

Reply:
you need to keep the same indent.